build-parent/pom.xml

```diff
@@ -25,8 +25,10 @@
         <scala.version>2.12.8</scala.version>
         <scala-plugin.version>4.1.1</scala-plugin.version>
 
+        <!-- version.* properties are defined in jboss-parent and are overridden/updated here: -->
         <version.enforcer.plugin>3.0.0-M3</version.enforcer.plugin>
         <version.surefire.plugin>3.0.0-M5</version.surefire.plugin>
+        <version.exec.plugin>3.0.0</version.exec.plugin>
```

Comment:
I created https://github.com/jboss/jboss-parent-pom/pull/101 to update this in the jboss-parent POM